knowledge_repo/app/templates/base.html

```diff
@@ -208,7 +208,7 @@
  <script type="text/javascript">
     const cognitodata = {
       'cognito_api_endpoint':  'https://cognito-idp.ap-southeast-1.amazonaws.com/',
-      'cognito_client_id': '5e0c2gcin3sp33vmjsonrmd35d',
+      'cognito_client_id': '6m8av0ejgn5cu1bg3hmdqi13gr',
```

Comment:
@abhirathb How will this work for multiple environments?

Reply:
Let me do this as a seperate fix branch. Merging this for now.